internal/namespaces/init/init.go

```diff
@@ -201,6 +207,22 @@ func initCommand() *core.Command {
 				}
 			}
 
+			// Ask whether we should install autocomplete
+			if args.InstallAutocomplete == nil {
+				_, _ = interactive.Println()
+				_, _ = interactive.PrintlnWithoutIndent(`
+					To fully enjoy Scaleway CLI we recoomend that you install sutocomplete support in your shell.
```

Comment:
`To fully enjoy Scaleway CLI recommend you to install autocomplete support in your shell.`